src/chart/graph/GraphView.js

```diff
@@ -202,6 +202,8 @@ export default echarts.extendChartView({
         var cy = data.getLayout('cy');
         var labelRotate = seriesModel.get('label.rotate') || 0;
         data.eachItemGraphicEl(function (el, idx) {
+            var itemModel = data.getItemModel(idx);
+            labelRotate = itemModel.get('label.rotate') || labelRotate;
```

Comment:
If `itemModel.get('label.rotate')` is set to be `0`, it will use `labelRotate`. But in fact, we wish it to be `0`.
It should be something like
```js
var itemRotate = itemModel.get('label.rotate');
if (itemRotate != undefined) {
    labelRotate = itemRotate;
}
```